src/Sources/Code/Module.cpp

```diff
@@ -556,23 +699,23 @@ void CALLBACK MyDispatchProc(SIMCONNECT_RECV* pData, DWORD cbData, void* pContex
 				str= m_str.get()->substr(prefix.length());
 				Client* newClient = RegisterNewClient(str);
 				SendNewClientResponse(client, newClient);
-				std::cout << "MobiFlight[" << client->Name.c_str() << "]: Received Client to register: " << str.c_str() << std::endl;				
+				std::cout << "MobiFlight[" << client->Name.c_str() << "]: Received Client to register: " << str.c_str() << std::endl;
 			}
 
 			if (m_str.get()->find("MF.Config.MAX_VARS_PER_FRAME.Set.") != std::string::npos) {
 				std::string prefix = "MF.Config.MAX_VARS_PER_FRAME.Set.";
 				str = m_str.get()->substr(prefix.length());
 				uint16_t value = static_cast<uint16_t>(std::stoi(str));
 				MOBIFLIGHT_MAX_VARS_PER_FRAME = value;
-				std::cout << "MobiFlight: Set MF.Config.MAX_VARS_PER_FRAME to " << value << std::endl;				
+				std::cout << "MobiFlight: Set MF.Config.MAX_VARS_PER_FRAME to " << value << std::endl;
 			}
 			break;
 		}
 
 		case SIMCONNECT_RECV_ID_EVENT_FRAME: {
 			SIMCONNECT_RECV_EVENT* evt = (SIMCONNECT_RECV_EVENT*)pData;
 			int eventID = evt->uEventID;
-
+			(void)eventID;
```

Comment:
what exactly does this void statement do?

Reply:
same as above

Reply:
ok. let's remove it then. thanks!